rfc/0054-sdk3-management-apis.md

```diff
@@ -3068,7 +3070,47 @@ Nothing
 
 ### URI
 
-    POST http://localhost:8091/pools/default/buckets/<bucket>/collections/<scope_name> -d name=<collection_name> -d maxTTL=<maxExpiry>
+    POST http://localhost:8091/pools/default/buckets/<bucket>/collections/<scope_name> -d name=<collection_name> -d maxTTL=<maxExpiry> -d history=<history>
+
+## UpdateCollection
+
+Creates a new collection.
+
+### Signature
+
+```
+void UpdateCollection(CollectionSpec collection, [options])
+```
+
+### Parameters
+
+* Required:
+
+  * `collection`: [`CollectionSpec`](#collectionspec) - specification of the collection.
+
+* Optional:
+
+  * `Timeout` or `timeoutMillis` (`int`/`duration`) - the time allowed for the operation to be terminated. This is controlled by the client.
+
+  * `maxExpiry` (`duration`) - Value for the max expiry of new documents created without an expiry.
```

Comment:
> Value for the max expiry of new documents created without an expiry.

<pendantry>
It also affects documents mutated with an expiry, by ensuring the document expiry is no farther in the future than the collection maxTTL.
</pedantry>